Author: Michael Poole <mdpoole@troilus.org>
Date:   Sun May 15 17:05:19 2011 -0400

    When IAuth provides creation timestamp, exclude it from the name.

diff -r 059a1d81b9fb ircd/s_auth.c
--- a/ircd/s_auth.c	Wed Jul 03 19:00:21 2013 +0200
+++ b/ircd/s_auth.c	Wed Jul 03 19:03:30 2013 +0200
@@ -1854,6 +1854,7 @@
     cli_user(cli)->acc_create = strtoul(params[0] + len + 1, &end, 10);
     if (*end == ':')
       cli_user(cli)->acc_id = strtoul(end + 1, NULL, 10);
+    params[0][len] = '\0';
   }
 
   /* Copy account name to User structure. */
